docs/ui/components/Sidebar/SidebarCollapsible.tsx

```diff
@@ -60,16 +61,16 @@ export function SidebarCollapsible(props: Props) {
 
   return (
     <>
-      <a css={titleStyle} onClick={toggleIsOpen}>
+      <ButtonBase css={titleStyle} aria-expanded={isOpen ? 'true' : 'false'} onClick={toggleIsOpen}>
         <div css={chevronContainerStyle}>
           <ChevronDownIcon
             size={iconSize.tiny}
             css={[chevronStyle, !isOpen && chevronClosedStyle]}
           />
         </div>
         <CALLOUT weight="medium">{info.name}</CALLOUT>
-      </a>
-      {isOpen && <div>{children}</div>}
+      </ButtonBase>
+      {isOpen && <div aria-hidden={!isOpen ? 'true' : 'false'}>{children}</div>}
```

Comment:
You don't need the `aria-hidden` because it's not rendered when hidden! 

I don't actually know how well this technique (unmounted the nodes) works with all assistive technology. Personally I often do:

```jsx
      {<div hidden={!isOpen}>{children}</div>}
``` 

This has the added benefit that the [hidden attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) works both for assistive and default tech (sighted users will also not be shown this), doesn't trash the DOM (removing/adding nodes), with the downside that the total DOM size is much bigger.

Best way would be to test with at least NVDA on Windows, and VoiceOver on Mac (and perhaps Android Talkback).

But if the unmount/mount is kept, you don't need to add `aria-hidden` :). Things you _could_ add (but optional because of the HTML structure) is `aria-controls` to the button where its value is the ID of this `div`.